Docker.DotNet/Microsoft.Net.Http.Client/UnixDomainSocketEndPoint.cs

```diff
@@ -0,0 +1,110 @@
+#if NETSTANDARD1_6
```

Comment:
OK, but now the copyright statement says read the LICENSE file, which is our license file, not the .NET one.